.changes/next-release/Feature-7eb562a6-6f2d-4b9c-8b70-59dc134cdbd5.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Added JSON validation for ECS task definition intellisense"
```

Comment:
@Paul-B-AWS can you provide some User documentation and some extension description revisions around this feature?
(Not in this PR)

Reply:
Yes, we are currently working on one.